lib/galaxy/datatypes/sequence.py

```diff
@@ -8,6 +8,7 @@
 import re
 import string
 import sys
+from __builtin__ import False
```

Comment:
Not sure - I didn't add it.  I just changed the import order so that flake8 would be happy.